### PR TITLE
add captureScreenshotAsBase64 and captureScreenshotAsList

### LINKED ISSUE #112: Add back captureScreenshotAsBase64
The sync webdriver provides both captureScreenshot and captureScreenshotAsBase64.

Atm:
```
  /// Take a screenshot of the current page as PNG.
  Stream<int> captureScreenshot() async* {
    var encoded = await getRequest('screenshot');
    yield* new Stream.fromIterable(CryptoUtils.base64StringToBytes(encoded));
  }
```
This could be split up again into two methods.

### PR DESCRIPTION
**Pull request by BlackHC**
This fixes #112 and #114

---
This change is [Reviewable](https://reviewable.io/reviews/google/webdriver.dart/115)

### CODE REVIEW

File: lib/src/web_driver.dart [hunk 1]
```diff
@@ -141,9 +141,18 @@ class WebDriver implements SearchContext {
   Mouse get mouse => new Mouse._(this);
 
   /// Take a screenshot of the current page as PNG.
```
Comment (goderbauer) [inline lib/src/web_driver.dart]: Can you update the doc comments of all three methods? It should outline the differences between the three screenshot methods.

File: lib/src/web_driver.dart [hunk 2]
```diff
@@ -141,9 +141,18 @@ class WebDriver implements SearchContext {
...
+  Future<String> captureScreenshotAsBase64() async =>
+      await getRequest('screenshot');
+
+  /// Take a screenshot of the current page as PNG.
+  Future<List<int>> captureScreenshotAsList() async {
+    var base64Encoded = captureScreenshotAsBase64();
+    return BASE64.decode(await base64Encoded);
+  }
+  /// Take a screenshot of the current page as PNG.
+
```
Comment (goderbauer) [inline lib/src/web_driver.dart]: remove empty line?

File: lib/src/web_driver.dart [hunk 3]
```diff
@@ -141,9 +141,18 @@ class WebDriver implements SearchContext {
...
   Stream<int> captureScreenshot() async* {
```
Comment (goderbauer) [inline lib/src/web_driver.dart]: What do you think about marking this method as deprecated (with a notice to redirect people to the new captureScreenshot as list)? As far as I can tell this method offers only disadvantages and no advantages. We should remove it in a future version of WebDriver.

**Comment by goderbauer** [comment]
Thanks for the contribution. Much appreciated. Please take a look at the comments I left.

**Comment by BlackHC** [comment]
Thanks. PTAL :)

---

Review status: 0 of 2 files reviewed at latest revision, 3 unresolved discussions.

---

*[lib/src/web_driver.dart, line 143 \[r1\]](https://reviewable.io:443/reviews/google/webdriver.dart/115#-KGkkaWInqZ-HjWF4O9B-r1-143:-KGkkoPAlSiEA7LN2KLp:-499146009) ([raw file](https://github.com/google/webdriver.dart/blob/aeedcf06dd9cf207d15724e2b30d7d38a47b4081/lib/src/web_driver.dart#L143)):*
Done.

---

*[lib/src/web_driver.dart, line 153 \[r1\]](https://reviewable.io:443/reviews/google/webdriver.dart/115#-KGkkaWInqZ-HjWF4O9B-r1-153:-KGkko-miUBPmzVmldeH:-499146009) ([raw file](https://github.com/google/webdriver.dart/blob/aeedcf06dd9cf207d15724e2b30d7d38a47b4081/lib/src/web_driver.dart#L153)):*
Done.

---

*[lib/src/web_driver.dart, line 154 \[r1\]](https://reviewable.io:443/reviews/google/webdriver.dart/115#-KGkkaWInqZ-HjWF4O9B-r1-154:-KGkkxWzdEwJE9eS4rWv:-499146009) ([raw file](https://github.com/google/webdriver.dart/blob/aeedcf06dd9cf207d15724e2b30d7d38a47b4081/lib/src/web_driver.dart#L154)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/webdriver.dart/115#-:-KGklc10fe5sUv0hMQNk:-14246560)*

**Comment by goderbauer** [comment]
LGTM.

I just kicked off another run on Travis. I'll merge if this one passes.

**Pull request merged**